tests/ci/worker/ubuntu_ami_for_ci.sh

```diff
@@ -75,6 +75,9 @@ cat <<EOT > /etc/docker/daemon.json
 }
 EOT
 
+# Increase the limit on number of virtual memory mappings to aviod 'Cannot mmap' error
+sysctl -w vm.max_map_count=2097152
```

Comment:
It's initial setup of OS image, so we need to preserve the config

```suggestion
echo "vm.max_map_count = 2097152" > /etc/sysctl.d/01-increase-map-counts.conf
```